src/core/server/http/http_server.ts

```diff
@@ -170,12 +173,9 @@ export class HttpServer {
     }
 
     await this.server.start();
+    const serverPath = this.config ? this.config.rewriteBasePath || this.config.basePath : '';
```

Comment:
optional: I think you could safely assume that `this.config` exists since `this.server` exists. Could change this to `this.config!.rewriteBasePath || this.config!.basePath`. If that ever fails, there's an internal logic bug in HttpServer.